reader_concurrency_semaphore.cc

```diff
@@ -919,7 +919,7 @@ future<> reader_concurrency_semaphore::execution_loop() noexcept {
             permit.on_executing();
             auto e = std::move(permit.aux_data());
 
-            tracing::trace(permit.trace_state(), "[reader concurrency semaphore] executing read");
+            tracing::trace(permit.trace_state(), "[reader concurrency semaphore: {}] executing read", _name);
```

Comment:
nit I would drop the colon and just make it `[reader concurrency semaphore {}]` (also below)

Reply:
applied